content/services/hosting/hosting_details.rst

```diff
@@ -9,45 +9,99 @@ offload work from your machine and/or infrastructure team. This list is not
+account on the system for us to use as needed for troubleshooting. We will not actively manage, monitor or back
+up these systems unless requested. If you run into an issue with your unmanaged system(s), we're still available to
+help of course!
+
 Virtualization
 --------------
 
-.. image:: /images/sysadmin.jpg
+.. image:: /images/student-sysadmin.jpg
     :scale: 100%
     :align: right
     :alt: Hosting Detail - Sysadmin
 
+We currently have two virtualization clusters available for use: `Ganeti`_ and `OpenStack`_. Depending on the project's
+use case, we host their VM(s) on either or both platforms. Both clusters are powered with the `KVM hypervisor`_.
+
+OpenStack
+^^^^^^^^^
+
+OpenStack offers a full API and web interface which allows projects to manage their VM resources as they see fit. Our
+OpenStack cluster currently offers the following services:
+
+- Compute (nova)
+- Cinder (cinder)
+- Image (glance)
+- Network (neutron)
+- Orchestration (heat)
+
+We plan to add more OpenStack services as needed by projects. Our cluster is currently running the Ocata release with
+eight compute nodes. Storage is powered via a six-node Ceph cluster. If you need to dynamically create/destroy/manage
+your services, our OpenStack cluster is the way to go.
+
+.. _OpenStack: http://openstack.org
+
 Ganeti
 ^^^^^^
 
-We offer virtual machines utilizing an open source stack consisting of `Ganeti`_ and the `KVM hypervisor`_. We have a
-cluster for small projects and offer VMs at a variety of sizes and platforms, all with full redundant storage. Ganeti
-is an Infrastructure as a Service (`IaaS`_) platform that we primarily use for long running VMs that need minimal
-changes. We provide minimal admin access to the VMs to project owners.
+We have used Ganeti for ten years and it continues to be the stable solution for long-running VMs that need minimal
+changes. We have a cluster with eight nodes for small projects and offer VMs at a variety of sizes and platforms, all
+with full redundant storage using DRBD. Ganeti is very simple to use and maintain; however, it doesn't provide a public
+API to deploy and manage the VMs. We do have a simple web management interface which allows projects to access the
+console and power the VM on and off as needed.
 
 .. _Ganeti: http://www.ganeti.org/
 .. _KVM hypervisor: http://www.linux-kvm.org/page/Main_Page
-.. _IaaS: https://en.wikipedia.org/wiki/Cloud_computing#Infrastructure_as_a_service_.28IaaS.29
 
-OpenStack
-^^^^^^^^^
+Build Farm Hosting
+------------------
+
+.. image:: /images/facebook-servers.jpg
+    :scale: 100%
+    :align: right
+    :alt: Hosting Detail - Build Farm hosting
 
-*ETA late 2017*
+Thanks to a hardware donation from Facebook in 2015, we have three OpenCompute Project (OCP) racks which contain a
+total of 90 OpenRack V2 "`Windmill`_" servers. These servers have 144 GB of RAM, 2 x Intel(R) Xeon(R) CPU E5-2660 0 @
```

Comment:
So these are called "Windmill" and not "Winterfell"? I thought "Winterfell" but it looks like that's a different model, so maybe I have it confused.

Reply:
Actually, you're correct. I'll fix this.